applications/tasks/src/kz_account_crawler.erl

```diff
@@ -184,7 +184,7 @@ crawl_account(AccountId) ->
     OpenResult = kz_datamgr:open_doc(?KZ_ACCOUNTS_DB, AccountId),
     check_then_process_account(AccountId, OpenResult).
 
--spec check_then_process_account(kz_term:ne_binary(), {'ok', kzd_accounts:doc()} | {'error',any()}) -> 'ok'.
+-spec check_then_process_account(kz_term:ne_binary(), kazoo_data:jobj_return()) -> 'ok'.
```

Comment:
I think this removes nuance when reading the spec - it isn't just a JSON object, it is the account doc. Specificity over generality in types is preferred.

Reply:
@jamesaimonetti That's true. But we can look at it in another way that it hides its true type behind an excessive description (it that a correct term? :). We did this excessive description in many places (not particularly doc/jobj) and makes developer more confused, it doesn't say what is really for example `doc` ? Is it `kz_json:object` or a record or a map or binaries or what? Sometimes we care about it we deep diving in the code and find out it is really jobj, but sometimes we assume and won't look it up.

Anyway ho, w about this:
```erlang
kazoo_data:jobj_return(kzd_accounts:doc())
```

Reply:
@jamesaimonetti Do you want me to close this or you're agree with change this to `kazoo_data:jobj_return(kzd_accounts:doc())`?